src/common/json_request.cpp

```
@@ -243,8 +235,9 @@ void EncryptJsonRequest::Parse(const std::string& request_body) {
     ParseCommon(request_body);
     
     // Load JSON for encrypt-specific fields
-    auto json_body = crow::json::load(request_body);
-    if (!json_body) return;
+    auto json_body_opt = SafeLoadJsonBody(request_body);
+    if (!json_body_opt) return;
```

similar to above, is this a silent failure?